iiif_prep/tests/test_prep.py

```diff
@@ -0,0 +1,46 @@
+from os.path import join
```

Comment:
The other thing you may want to test (which would be pretty easy) would be the `get_mezzanine_path` function.